__tests__/integration/core-api/handlers/businesses.test.ts

```diff
@@ -60,10 +67,12 @@ describe("API 2.0 - Businesses", () => {
 
     describe("GET /businesses/:id", () => {
         it("should GET a business by the given valid identifier", async () => {
-            const response = await utils.request("GET", `businesses/${publicKey}`, { transform: false });
-            expect(response).toBeSuccessfulResponse();
-
-            expect(response.data.data.attributes.business).toEqual(businessAttribute);
+            for (const identifier of Object.values(validIdentifiers)) {
```

Comment:
Can you instead of this loop, use `it.each`, you could do something like
```
it.each(Object.entries(validIdentifiers))("should GET a business by %s : %s", async (idName, idValue) => {
```
it then shows up better in the test recap (if one of address, public key or username fails you will see directly which one fails)